utils/setup.cfg

```diff
@@ -7,10 +7,12 @@ universal=1
 [nosetests]
 tests=../,../roles/openshift_master_facts/test/,test/
 verbosity=2
-with_coverage=1
-cover_html=1
-cover_package=ooinstall
-cover_min_percentage=70
+with-coverage=1
+cover-html=1
+cover-inclusive=1
+cover-min-percentage=70
+cover-erase=1
```

Comment:
WRT: `cover-erase=1` personally I prefer to run coverage with the previous results retained. This allows you to easily see if coverage has increased or decreased in any given PR/commit. I understand why you'd set the value to `1` If that doesn't work on travis (I imagine travis erases all test environments between runs).